test/SignerBondsUniswapV2.test.js

```diff
@@ -74,6 +74,181 @@ describe("SignerBondsUniswapV2", () => {
     })
   })
 
+  describe("setMaxAllowedPriceImpact", () => {
+    const maxAllowedPriceImpact = 10000
+
+    context("when caller is not the governance", () => {
+      it("should revert", async () => {
+        await expect(
+          signerBondsUniswapV2
+            .connect(thirdParty)
+            .setMaxAllowedPriceImpact(maxAllowedPriceImpact)
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when max allowed price impact too big", () => {
```

Comment:
nitpick: is too high?

Reply:
Fixed in https://github.com/keep-network/coverage-pools/pull/156/commits/7293ab108d709dce06e5db22230eb2d5ad52d647